lms/static/sass/features/_course-sock.scss

```diff
@@ -0,0 +1,172 @@
+.verification-sock {
+  display: inline-block;
+  position: relative;
+  width: 100%;
+  margin-top: $baseline;
+  max-width: 1180px;
+  margin: $baseline auto 0;
+
+  .action-toggle-verification-sock {
+    @include left(50%);
+    @include margin-left(-1 * $baseline * 15/2);
+    position: absolute;
+    top: (-1 * $baseline);
+    width: ($baseline * 15);
+    color: $button-bg-hover-color;
+    background-color: $success-color;
+    border-color: $success-color;
+    background-image: none;
+    box-shadow: none;
+    -webkit-transition: background-color 0.5s;
+    transition: background-color 0.5s;
+
+    &.active{
+      color: $success-color;
+      background-color: $button-bg-hover-color;
+      border-color: $success-color;
+      background-image: none;
+      box-shadow: none;
+
+      &:hover {
+        color: $button-bg-hover-color;
+        background-color: $success-color-hover;
+        border-color: $success-color-hover;
+        background-image: none;
+        box-shadow: none;
+      }
+    }
+
+    &:hover {
+      color: $button-bg-hover-color;
+      background-color: $success-color-hover;
+      border-color: $success-color-hover;
+      background-image: none;
+      box-shadow: none;
+    }
+  }
+
+  .verification-main-panel {
+    display: none;
+    overflow: hidden;
+    border-top: 1px solid $lms-border-color;
+    padding: ($baseline * 5/2) ($baseline * 2);
+
+    .verification-desc-panel {
+      color: $black-t3;
```

Comment:
Can you make this a variable. I'm not sure that $black-t3 is defined in the pattern library.